modules/gapi/perf/common/gapi_render_perf_tests.cpp

```diff
@@ -0,0 +1,1238 @@
+    cv::gapi::wip::draw::Prims prims;
+
+    // Rects
+    cv::Rect rect(200, 200, 200, 200);
+    int shift = 0;
+
+    for (int i = 0; i < rects_num; ++i)
+        prims.emplace_back(cv::gapi::wip::draw::Rect(rect, color, thick, lt, shift));
+
+    // Mosaic
+    cv::Rect mos(200, 200, 200, 200);
+    int cellsz = 25;
+    int decim = 0;
+
+    for (int i = 0; i < rects_num; ++i)
+        prims.emplace_back(cv::gapi::wip::draw::Mosaic(mos, cellsz, decim));
+
+    // Text
+    std::string text = "Some text";
+    cv::Point org(200, 200);
+    int ff = FONT_HERSHEY_SIMPLEX;
+    double fs = 2.0;
+    bool blo = false;
+
+    for (int i = 0; i < text_num; ++i)
+        prims.emplace_back(cv::gapi::wip::draw::Text(text, org, ff, fs, color, thick, lt, blo));
+
+    // Image
+    cv::Rect rect_img(100, 100, 100, 100);
+    double transparency = 1.0;
+
+    cv::Mat img(rect_img.size(), CV_8UC3, color);
+    cv::Mat alpha(rect_img.size(), CV_32FC1, transparency);
+    auto tl = rect_img.tl();
+    cv::Point org_img = {tl.x, tl.y + rect_img.size().height};
+
+    for (int i = 0; i < image_num; ++i)
+        prims.emplace_back(cv::gapi::wip::draw::Image(org_img, img, alpha));
+
+    cv::GMat in;
+    cv::GArray<cv::gapi::wip::draw::Prim> arr;
+
+    cv::GComputation comp(cv::GIn(in, arr),
+                          cv::GOut(cv::gapi::wip::draw::render3ch(in, arr)));
+
+    // Warm-up graph engine:
+    comp.apply(gin(gapi_mat, prims), gout(gapi_mat));
+
+    TEST_CYCLE()
+    {
+        comp.apply(gin(gapi_mat, prims), gout(gapi_mat));
+    }
+
+    // OpenCV code //////////////////////////////////////////////////////////////
+    {
+        cv::rectangle(ref_mat, rect, color, thick, lt, shift);
+
+        drawMosaicRef(ref_mat, mos, cellsz);
+
+        cv::putText(ref_mat, text, org, ff, fs, color, thick, lt, blo);
+
+        blendImageRef(ref_mat, org_img, img, alpha);
+    }
+
+    // Comparison //////////////////////////////////////////////////////////////
+    {
+        EXPECT_EQ(0, cv::norm(gapi_mat, ref_mat, NORM_INF));
+    }
+
+    SANITY_CHECK_NOTHING();
+}
+
+PERF_TEST_P_(RenderTestPolyItems, RenderPolyItemsPerformanceNV12OCVTest)
+{
+    int thick = 2;
+    int lt = LINE_8;
+    cv::Scalar color(100, 50, 150);
+
+    cv::Size sz;
+    int rects_num, text_num, image_num;
+    std::tie(sz, rects_num, text_num, image_num) = GetParam();
+
+    cv::Mat y_ref_mat, uv_ref_mat;
+
+    cv::Mat y_in_gapi_mat(sz, CV_8UC1), uv_in_gapi_mat(sz / 2, CV_8UC2),
+        y_out_gapi_mat(sz, CV_8UC1), uv_out_gapi_mat(sz / 2, CV_8UC2);
+
+    create_rand_mats(sz, CV_8UC1, y_ref_mat, y_in_gapi_mat);
+    create_rand_mats(sz / 2, CV_8UC2, uv_ref_mat, uv_in_gapi_mat);
+
+    // G-API code //////////////////////////////////////////////////////////////
+    cv::gapi::wip::draw::Prims prims;
+
+    // Rects
+    cv::Rect rect(200, 200, 200, 200);
+    int shift = 0;
+
+    for (int i = 0; i < rects_num; ++i)
```

Comment:
It's better to use brackets even if `for` consist of only one line

Reply:
Fixed